source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3867,9 +3925,15 @@ export abstract class Action extends CardObject {
         }
 
         if (this.title) {
+            buttonElement.setAttribute("aria-label", this.title);
             titleElement.innerText = this.title;
         }
 
+        if (this.tooltip) {
+            buttonElement.setAttribute("aria-description", this.tooltip);
```

Comment:
This came as an action item out of the accessibility review:

On a standalone `Action` button (in an `ActionSet` or `AdaptiveCard.actions`), if there is an `iconUrl` and `tooltip`, but no `title`, we wanted to promote the tooltip to `aria-label` (or its equivalent on other platforms). This shouldn't apply to `selectAction`s. I think the reasoning was that the content has no text alternative, so we need something to be the label.

@RebeccaAnne, correct me if I remember incorrectly

Reply:
Hm I have no recollection of that. But to the end user, the result should be the same whether tooltip is set as `aria-label` or `aria-description` so I'm not sure there is any change to make here.

Reply:
According to the aria spec, Accessible Name doesn't include the description, so the control would still have no accessible name (so, it would likely be flagged by tools like Accessibility Insights). This would be harder to push back on because it doesn't follow the spec and doesn't work in Narrator.

I am not strongly opposed to leaving it as is, but it is just something we said at the review.

Reply:
You're correct. I'll make the change.

Reply:
Fixed